modules/update-evaluate-availability.adoc

```diff
@@ -0,0 +1,114 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding_updates/how-updates-work.adoc
+
+:_content-type: CONCEPT
+[id="update-evaluate-availability_{context}"]
+= Evaluating update availability
+
+The Cluster Version Operator (CVO) periodically queries the OpenShift Update Service (OSUS) for the most recent data about update possibilities.
+This data is based on the channel to which the cluster is subscribed.
+The CVO then saves information about update recommendations and risks into either the `availableUpdates` or `conditionalUpdates` field of its ClusterVersion resource.
+
+The CVO periodically checks the conditional updates for update risks.
+These risks are conveyed through the data served by the OpenShift Update Service, which contains information for each version about known issues that may affect a cluster updated to that version.
+Most risks are limited to clusters with specific characteristics, such as clusters with a certain size or clusters that are deployed in a particular cloud platform.
+
+The CVO continuously evaluates its cluster characteristics against the conditional risk information for each conditional update, and if it finds that the cluster matches the criteria, it stores this information in the `conditionalUpdates` field of its ClusterVersion resource.
+If the CVO finds that the cluster doesn't match the risks of an update, or that there are no risks associated with the update, it stores the target version in the `availableUpdates` field of its ClusterVersion resource.
+
+The user interface, either the web console or the OpenShift CLI (`oc`), then presents this information to the administrator.
+Each "supported but not recommended" update recommendation contains a link to further resources about the risk so that the administrator can make an informed decision about the update.
```

Comment:
> "supported but not recommended"

Do we use quotes like this in the docs? Should it be italics or bold?

Reply:
@dfitzmau I suppose it can go either way and work fine, I think there are a few instances of "you" in this doc that relate to specific commands a user could use.

However, this page is all purely conceptual info (with exceptions for commands users could use to find real examples of conceptual stuff being explained), and the IBM style guide does seem to permit the use of [Third person](https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs#third-person) when describing "concepts, facts, and results." I'm personally inclined to keep the focus on the concept and not the user here, but let me know if you disagree.

Reply:
@mburke5678 I don't think this is an appropriate use for quotes now that I think about it. IIRC, I think it was an attempt by QE and/or devs to highlight that this is approximate wording in our CLI output and web UI, but without explicitly stating this as the exact wording (in case it changes down the line).

I think I'll take the risk of maintenance down the line and use bold formatting, since quotes are probably not acceptable here